docs/docs/dev/other-specifications/publishers/s3.md

```diff
@@ -13,7 +13,11 @@ Bacalhau's S3 Publisher provides users with a secure and efficient method to pub
 - **Key** `(string: <required>)`: The object key within the specified bucket where the task results will be stored.
 - **Endpoint** `(string: <optional>)`: The endpoint URL of the S3 service (useful for S3-compatible services).
 - **Region** `(string: <optional>)`: The region where the S3 bucket is located.
+- **Compress** `(bool: false)`: Indicates whether the task results should be compressed before storage. Only compressed objects can be downloaded using the `bacalhau get <job_id>` command.
 
+## Publishing Flexibility
+
+The S3 Publisher is adept at handling both individual files and full directories. Each file within a directory is uploaded as a separate S3 object. If the `Compress` option is enabled, the entire directory is compressed into a single object, enhancing efficiency and reducing storage requirements. Only compressed objects can be downloaded using the `bacalhau get <job_id>` command.
```

Comment:
@wdbaruni can you comment if this is still true? If I remember correctly we always compress now.

Reply:
Compress is not part of the spec: 
https://github.com/bacalhau-project/bacalhau/blob/main/pkg/s3/types.go#L49